addons/storysource/src/StoryPanel.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { API } from '@storybook/api';
+import { styled } from '@storybook/theming';
+import { Link } from '@storybook/router';
+import {
+  SyntaxHighlighter,
+  SyntaxHighlighterProps,
+  SyntaxHighlighterRendererProps,
+} from '@storybook/components';
+
+import createElement from 'react-syntax-highlighter/dist/esm/create-element';
```

Comment:
breaks `@storybook/components` abstraction?

Reply:
fixing - adding export to `@storybook/components`